packages/api/tests/endpoints/test-executions.js

```diff
@@ -57,6 +71,7 @@ test.before(async () => {
   // create fake execution records
   fakeExecutions.push(fakeExecutionFactory('completed'));
   fakeExecutions.push(fakeExecutionFactory('failed', 'workflow2'));
+  // TODO - this needs updated after postgres->ES work
```

Comment:
Oh because the source of the ES index needs to be PG. Do we have that work planned?

Reply:
Yes, there's a 'index postgres writes to ES' ticket, if that's what you mean.

Reply:
Awesome. Yup! That's what I was after.